lib/experimental/Navigation/Sidebar/CompanySelector/index.tsx

```diff
@@ -18,13 +19,23 @@ export type CompanySelectorProps = {
   selected?: string
   onChange: (value: string) => void
   isLoading?: boolean
+  notification?: boolean
```

Comment:
maybe `withNotification`? or `withBadge`? 

`notification` hints me I can provide a message, which I cannot

Reply:
Yes! I went with `withNotification` because using `badge` might imply that any badge can be used, but in this case, it's specifically tied to the notification circle icon.